src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleEqualityTests.cs

```diff
@@ -2835,12 +2895,9 @@ int M(out int x)
 ";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
-                // (7,14): error CS0841: Cannot use local variable 'z' before it is declared
-                //         _ = (z, M(out int z)) == (1, 2); // error
-                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "z").WithArguments("z").WithLocation(7, 14),
-                // (7,13): error CS0019: Operator '==' cannot be applied to operands of type 'var' and 'int'
```

Comment:
Why did this warning get removed?

Reply:
Because it is a cascaded diagnostic and a bad one at that.  `var` refers to an error type.  There is no `var` in the source at all!

---
In reply to: [301726282](https://github.com/dotnet/roslyn/pull/37052#discussion_r301726282) [](ancestors = 301726282)